onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -1893,9 +1920,7 @@ including arg name, arg type (contains both type and shape).)pbdoc")
             }
             res << ")";
 
-            return std::string(res.str());
-          },
-          "converts the node into a readable string")
+            return std::string(res.str()); }, "converts the node into a readable string")
```

Comment:
nit: it looks slightly odd for the closing brace of a multi-line function to not be on its own line. if this is clang-format's preference I can live with it.

Reply:
it's all clang-format